packages/react-app/.eslintrc.js

```diff
@@ -2,31 +2,10 @@ module.exports = {
   env: {
     browser: true,
   },
-  parser: "babel-eslint",
-  extends: [/*"airbnb"*/ "plugin:prettier/recommended", "plugin:react/recommended", "plugin:@next/next/recommended"],
-  plugins: [
-    /*"babel"*/
-  ],
+  extends: ["eslint:recommended", "prettier", "next"],
   rules: {
-    "prettier/prettier": "warn",
-    "prettier/prettier": [
-      "warn",
-      {
-        endOfLine: "auto",
-      },
-    ],
-    "import/prefer-default-export": "off",
-    "prefer-destructuring": "off",
-    "prefer-template": "off",
-    "react/prop-types": "off",
-    "react/destructuring-assignment": "off",
-    "no-console": "off",
-    "jsx-a11y/accessible-emoji": ["off"],
-    "jsx-a11y/click-events-have-key-events": ["off"],
-    "jsx-a11y/no-static-element-interactions": ["off"],
-    "no-underscore-dangle": "off",
-    "no-nested-ternary": "off",
-    "no-restricted-syntax": "off",
-    "no-plusplus": "off",
+    "no-console": "warn",
+    "no-undef": "warn",
```

Comment:
This should be an error as well.

Reply:
the thing about `no-undef` is that it even complains about `Promise` being undefined, which is a global

Reply:
@Dhaiwat10 Yeah, that can be resolved by adding the `es6` parameter like this:
```
  env: {
    browser: true,
    es6: true
  },
 ```

Reply:
Cool. What about the console.logs? I don't think throwing an error for them is a good idea

Reply:
Everything else is okay, just for `no-undef` :)

Reply:
Done